_data/people.yml

```
@@ -232,3 +232,8 @@
   homepage: https://vivian-liu.com/
   image: /files/images/avatar/vivian_liu.png
   category: Student
+
+- name: Tao Long
+  homepage: https://iamtaolong.github.io/
+  image: /files/images/avatar/tao.jpeg
```

Can you rename the photo to your full name, to match the existing student photo format?
```suggestion
  image: /files/images/avatar/tao_long.jpeg
```
Thanks!